esti/commit_test.go

```diff
@@ -23,7 +23,7 @@ func TestCommitSingle(t *testing.T) {
 
 			_, objContent := uploadFileRandomData(ctx, t, repo, mainBranch, objPath, direct)
 			commitResp, err := client.CommitWithResponse(ctx, repo, mainBranch, api.CommitJSONRequestBody{
-				Message: "nessie:singleCommit",
+				Message: "esti:singleCommit",
```

Comment:
Suggest defining the executor name as a constant in the package scope and using it in all string references.

Reply:
Remove the "esti" name from the code